packages/miniapp-utils/src/helpers/i18n.tsx

```
@@ -0,0 +1,479 @@
+> = LocaleSelection<AvailableLocale> & { messages: MessagesShape }
+
+type SSRResult<PropsType extends Record<string, unknown>> = {
+    props: PropsType
+}
+type SSRResultWithI18N<
+    AvailableLocale extends string,
+    MessagesShape extends Record<string, string>,
+    PropsType extends Record<string, unknown>,
+> = {
+    props: PropsType & {
+        [I18N_SELECTED_LOCALE_PROP_NAME]?: AvailableLocale
```

Biome (1.9.4)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)